src/components/select/select.js

```diff
@@ -175,15 +186,14 @@ function SelectDirective($mdSelect, $mdUtil, $mdTheming, $mdAria, $interpolate,
         }
       }
 
-
       var originalRender = ngModelCtrl.$render;
-      ngModelCtrl.$render = function() {
+      ngModelCtrl.$render = function () {
```

Comment:
I did run an auto-formatter. I think we need a team formatter; for WebStorm and more.